web/js/codeworld_shared.js

```diff
@@ -1203,4 +1203,19 @@ function printMessage(type, message) {
     }
 
     outputDiv.scrollTop = outputDiv.scrollHeight;
+
+    if (isRuntime && type === "error") {
```

Comment:
What about updating the error in the output window?  That's actually more important than updating the linter, I think.

Reply:
I think output window is kind of terminal output. So updating of already printed line is weird for me. 

We can update existing line, or delete old one and print new at the end. What is better?

Reply:
I think we should update the existing line.  I realize that seems weird if you think of the output window as a terminal, but:
1. I don't think my students really do think of it that way.  In fact, I don't think they usually have any notion of what a terminal is.
2. As more and more errors are deferred, we end up in a weird position where most non-parse error messages are printed twice.  That's obviously not what you would want.

In this case, you have essentially two of the same message, differing only in whether it's a warning or an error.  It makes sense to just change the warning to an error, rather than appending it at the end.

Reply:
This comment is subsumed by the discussion below, so I'll resolve it.  The plan is currently to render with error styling from the beginning, but then append a message when the runtime crash occurs saying: "Giving up because of the error at program.hs:10:42"